tensorflow/tools/docker/Dockerfile.devel-gpu-cuda9-cudnn7

```diff
@@ -19,6 +19,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         pkg-config \
         python-dev \
         python-pip \
+        python-wheel \
```

Comment:
Instead let's add `wheel` to the big pip install command below, that way we can get a more up-to-date version than what ubuntu can give us.

Reply:
Agree.